src/stabilization_guide.md

```diff
@@ -0,0 +1,181 @@
+
+# Request for stabilization
+
+    Once an unstable feature has been well-tested with no outstanding
+    concern, anyone may push for its stabilization. It involves the
+    following steps.
+
+    - Documentation PRs
+    - Write a stabilization report
+    - FCP
+    - Stabilization PR
+
+## Documentation PRs
+
+    Prepare PRs to update documentations involing this new feature.
+    You need to submit PRs for  repositories The Reference, The Book
+    and Rust by Example.
+    Maintainers of these repositories will keep these PRs open until
+    the whole stabilization process has completed. Meanwhile, we can
+    proceed to the next step.
+
+## Write a stabilization report
+
+    Find the tracking issue of the feature, and create a short
+    stabilization report. Essentially this would be a brief summary
+    of the feature plus some links to test cases showing it works
+    as expected, along with a list of edge cases that came up and
+    and were considered. This is a minimal "due diligence" that
+    we do before stabilizing.
+
+    The report should contain:
+
+        - A summary, showing examples (e.g. code snippets) what is
+          enabled by this feature.
+        - Links to test cases in our test suite regarding this feature
+          and describe the feature's behavior on encountering edge cases.
+        - Links to the documentations (the PRs we have made in the
+          previous steps).
+        - Any other relevant information(Examples of such reports can
+          be found in rust-lang/rust#44494 and rust-lang/rust#28237).
+
+## FCP
+
+If any member of the team responsible for tracking this
+feature agrees with stabilizing this feature, they will
+start the FCP (final-comment-period) process by
+commenting
+
+    ```bash
+    @rfcbot fcp merge
+    ```
+
+The rest of the team members will review the proposal. If the final
+decision is to stabilize, we proceed to do the actual code modification.
+
+## Stabilization PR
+
+Once we have decided to stabilize a feature, we need to have a PR that
+actually makes that stabilization happen. These kinds of PRs are a
+great way to get involved in Rust, as they take you on a little tour
+through the source code.
+
+Here is a general guide to how to stabilize a feature -- every feature
+is different, of course, so some features may require steps beyond 
+what this guide talks about.
+
+Note: Before we stabilize any feature, it is rule that it should appear
+in the documentation.
+
+### Updating the feature-gate listing
+
+There is a central listing of feature-gates in
+`src/libsyntax/feature_gate.rs`. Search for the `declare_features!`
+macro. There should be an entry for the feature you are aiming to
+stabilize, something like (this example is taken from
+[rust-lang/rust#32409]:
+
+    ```
```

Comment:
```suggestion
    ```rust
```

Reply:
Also, please un-indent this (also applies elsewhere)

Reply:
done